src/main/java/haveric/recipeManager/recipes/CraftRecipeParser.java

```diff
@@ -88,7 +88,7 @@ public boolean parseRecipe(int directiveLine) throws Exception {
 
         recipe.setIngredients(ingredients); // done with ingredients, set 'em
 
-        if (recipe.hasFlag(FlagType.REMOVE)) {
+        if (recipe.hasFlag(FlagType.REMOVE) && !Version.has1_12Support()) {

```

Comment:
Without the NMS stuff, probably there is no need for it. For the NMS stuff, if the recipes lacked outcomes it failed during some other part of the matching routines. Probably just cruft at this point.